### ISSUE TITLE
Upload process does not appear to queue upload and fails when multiple files have been uploaded

### ISSUE THREAD

**Issue by BethRR** [winwinwiki/winwin-frontend#484]
User has a series of files to upload to avoid creating issues by loading too large a file. They start one job and then upload another file to be queued.

Expected that the system would queue the jobs and then run them as feasible. 
Actual result is that all jobs failed (and no notification was sent about the failure #483). 

IF you are looking in logs, both jobs were started around 11am on 7/19/19. They were each 1000k and conformed to the template.

**Comment by arvindkhatik012** [comment]
Hi John,Beth
we tried to replicate the issue and uploaded the two files each with 1k organizations at the same time and it worked fine ,hence marking the issue as fixed.

Click the below link to Check the success notifications and time stamp. 
https://winwin2016.slack.com/archives/CL45P64BA/p1563613816000400
https://winwin2016.slack.com/archives/CL45P64BA/p1563613852000700

Click the below link for your reference.
https://winwin2016.slack.com/archives/CL45P64BA/p1563614775006000

**Comment by BethRR** [comment]
Seems to have been fixed

**Comment by BethRR** [comment]
Mistakenly closed this. Testing on new staging environment.

**Comment by BethRR** [comment]
@arvindkhatik012  -- I tested on the new staging environment. I loaded the two files in the attached zip file, one right after the other. The files seem to be stuck in limbo. The first file is 1.2mb with has 7869 rows and the second file is only 6kb with 2398 rows. 

Zip file containing test files: [UploadSequencedTest.zip](https://github.com/winwinwiki/winwin-frontend/files/3469546/UploadSequencedTest.zip)
